### PR TITLE
Feature/compare trainings

### PR DESCRIPTION
**Pull request by ofahchouch-gh**
updated: training comparison, changed butterfly chart to bart chart and added a relative & absolute mode

### CODE REVIEW

**Comment by StanBankras** [comment]
If there is no training to compare with, there is not really a use to show the same bars twice:

compare

**Comment by StanBankras** [comment]
Other than that, the comparing graph works as expected! 👍

File: www/src/components/molecules/PersonalDetails.vue [hunk 1]
```diff
@@ -27,16 +27,22 @@
...
   computed: {
     selectedPatient() {
-      return this.$store.getters.selectedPatient;
+      return this.getSelectedPatient();
     }
   },
   watched: {},
-  methods: {}
+  methods: {
+    getSelectedPatient() {
```
Comment (StanBankras) [inline www/src/components/molecules/PersonalDetails.vue:41]: I would recommend moving functionality like this outside of components, and instead pass a prop with this data from the parent component. If you do this in the component itself, it's not reusable anymore.

File: www/src/views/Results.vue [hunk 2]
```diff
@@ -56,32 +56,33 @@ export default {
...
-    return {
-      updateCurrentTrainingEndpoint: '/api/training',
-      notes: undefined,
-      id: null
-    }
-  },
-  mounted() {
-    this.id = this.$route.params.id;
+  watch: {
+    '$store.state.trainings': (value) => {
```
Comment (StanBankras) [inline www/src/views/Results.vue:83]: This could be a computed variable instead of a watch, it would be cleaner and more efficient I think.

**Comment by ofahchouch-gh** [comment]
@StanBankras thanks for the feedback. Appreciate it. Have rewritten the code according to the feedback.

**Pull request merged**